src/debug_wait.c

```diff
@@ -108,6 +108,23 @@ ts_debug_waitpoint_wait(DebugWait *waitpoint, bool blocking)
 	ereport(DEBUG3, (errmsg("proceeding after waitpoint '%s'", waitpoint->tagname)));
 }
 
+void
+ts_debug_waitpoint_error_if_enabled(DebugWait *waitpoint)
```

Comment:
It's generally a good idea to have at least minimal comment for the non-static procedure.